pkg/controller/expcertificates/trigger/policies/policies_test.go

```diff
@@ -455,18 +444,15 @@ func TestDefaultPolicyChain(t *testing.T) {
 					corev1.TLSPrivateKeyKey: staticFixedPrivateKey,
 					corev1.TLSCertKey: selfSignCertificateWithNotBeforeAfter(t, staticFixedPrivateKey,
 						&cmapi.Certificate{Spec: cmapi.CertificateSpec{CommonName: "example.com"}},
-						time.Now().Add(time.Minute*-45),
-						// expires in 15 minutes time
-						time.Now().Add(time.Minute*15),
+						clock.Now().Add(time.Minute*-30),
+						// expires in 5 minutes time
+						clock.Now().Add(time.Minute*5),
```

Comment:
I'm losing concentration now, it's soooo hot today, but these tests all make sense, I think.